src/Database/Attach/File.php

```diff
@@ -93,14 +98,10 @@ class File extends Model
 
     /**
      * Creates a file object from a file an uploaded file.
-     * @param Symfony\Component\HttpFoundation\File\UploadedFile $uploadedFile
+     * @param \Symfony\Component\HttpFoundation\File\UploadedFile $uploadedFile
      */
     public function fromPost($uploadedFile)
     {
-        if ($uploadedFile === null) {
-            return;
-        }
-
```

Comment:
What was the reason for this change?

Reply:
`$uploadedFile` should not be null, according to both the docblock and the signature. If it is, this code block is silently discarding it, when it should throw an exception.